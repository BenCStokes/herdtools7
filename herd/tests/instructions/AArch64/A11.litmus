AArch64 A11

(* Tests strb (register), UXTW - overflow on 8 bit store gives value 0 as expected*)
(* Same as A09 but with a bump on the value in X0 *)
{ int8_t 0:X0 = 256; 0:X1=x; uint64_t x; 0:X2=0; }

P0;
  STRB W0, [X1, X2, UXTW];

exists (x=0)
