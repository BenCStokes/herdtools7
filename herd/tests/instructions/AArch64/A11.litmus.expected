Test A11 Allowed
States 1
[x]=0;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists ([x]=0)
Observation A11 Always 1 0
Hash=a8c6534df7dbf0ba0b8e146aaf76977d

